src/js/constants/ContainerConstants.js

```diff
@@ -1,22 +1,22 @@
 const ContainerConstants = {
-  NETWORK: {
+  network: Object.freeze({
     BRIDGE: "BRIDGE",
     HOST: "HOST"
-  },
-  PORTMAPPINGS: {
-    PROTOCOL: {
+  }),
+  portMappings: {
```

Comment:
I think that is right. It's a property of the object:  `ContainerConstants.portMappings` (cfr. `SomeReactComponent.displayName`)